components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/NetworkMessageEncoder.cs

```diff
@@ -89,9 +97,8 @@ public class NetworkMessageEncoder : IMessageEncoder {
             var current = notifications.GetEnumerator();
             var processing = current.MoveNext();
             var messageSize = 2; // array brackets
-            maxMessageSize -= 2048; // reserve 2k for header
             var chunk = new Collection<NetworkMessage>();
```

Comment:
why do we reserve 2k?

Reply:
According to @koepalex for the PubSub header. We decided to leave this buffer and consult with Christian next week. It might not be necessary for Samples.